Gemfile

```diff
@@ -13,7 +13,11 @@ group :test do
   gem 'httpclient', '>= 2.2'
   gem 'mime-types', '~> 1.25', :platforms => [:jruby, :ruby_18]
   gem 'minitest', '>= 5.0.5'
-  gem 'net-http-persistent', '~> 2.9.4'
+  if RUBY_VERSION >= '2.1'
```

Comment:
http://bundler.io/v1.13/man/gemfile.5.html#PLATFORMS seems not to support this use-case.

http://bundler.io/v1.13/man/gemfile.5.html#INSTALL_IF is perhaps a candidate?